tests/unit/test_triton_inference.py

```diff
@@ -6,10 +6,9 @@
 import nvtabular as nvt
 import nvtabular.ops as ops
 
-# triton = pytest.importorskip("nvtabular.inference.triton")
+triton = pytest.importorskip("nvtabular.inference.triton")
 
 
-@pytest.mark.skip(reason="not playing nice with others")
 @pytest.mark.parametrize("engine", ["parquet"])
 def test_generate_triton_model(tmpdir, engine, df):
     import nvtabular.inference.triton as triton
```

Comment:
remove this import I added for the flake8 stuff. so triton mention below would still be tied to something... with uncommenting above we don't need to have this import here

Reply:
done